symphony-bdk-spring/symphony-bdk-app-spring-boot-starter/src/main/java/com/symphony/bdk/app/spring/auth/CircleOfTrustController.java

```diff
@@ -0,0 +1,103 @@
+package com.symphony.bdk.app.spring.auth;
+
+import com.symphony.bdk.app.spring.auth.model.AppInfo;
+import com.symphony.bdk.app.spring.auth.model.AppToken;
+import com.symphony.bdk.app.spring.auth.model.JwtInfo;
+import com.symphony.bdk.app.spring.auth.model.TokenPair;
+import com.symphony.bdk.app.spring.auth.model.UserId;
+import com.symphony.bdk.app.spring.exception.AppAuthException;
+import com.symphony.bdk.app.spring.exception.InvalidJwtException;
+import com.symphony.bdk.app.spring.exception.InvalidTokenException;
+import com.symphony.bdk.core.auth.AppAuthSession;
+import com.symphony.bdk.core.auth.ExtensionAppAuthenticator;
+import com.symphony.bdk.core.auth.exception.AuthInitializationException;
+import com.symphony.bdk.core.auth.exception.AuthUnauthorizedException;
+import com.symphony.bdk.core.auth.jwt.UserClaim;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.validation.Valid;
+
+@Slf4j
+@RestController
+@RequestMapping("/bdk/v1/app")
+public class CircleOfTrustController {
+
+  @Value("${bdk.app.auth.jwtCookie.enabled:false}")
+  private Boolean jwtCookieEnable;
+
+  @Value("${bdk.app.auth.jwtCookie.maxAge:86400}")
+  private Integer jwtCookieMaxAgeInSec;
```

Comment:
This is a bad practice, this starter should provide its own properties, like [SymphonyBdkCoreProperties](https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/blob/master/symphony-bdk-spring/symphony-bdk-core-spring-boot-starter/src/main/java/com/symphony/bdk/spring/SymphonyBdkCoreProperties.java). 

So that you could directly inject a `SymphonyBdkAppProperties` in your class.